sdp/processors/langs/armenian.py

```diff
@@ -0,0 +1,81 @@
+import os
+import pandas as pd
+from pathlib import Path
+from sdp.processors.base_processor import BaseProcessor, BaseParallelProcessor, DataEntry
+from sdp.utils.common import load_manifest
+
+
+class GetSourceFolder(BaseParallelProcessor):
+    """
+    Processor for extracting source folder from file paths and updating the manifest.
+
+    Args:
+        source_filepath (str): The field containing the file path in the manifest.
+        source_field (str): The field to store the extracted source folder in the manifest.
+        **kwargs: Additional keyword arguments to be passed to the base class `BaseParallelProcessor`.
+
+    Note:
+        This class inherits from the `BaseParallelProcessor` class and extends its functionality to extract source information from file paths and update the manifest.
+    """
+    def __init__(
+        self,
+        source_filepath: str,
+        source_field: str,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.input_field = source_filepath
+        self.output_field = source_field
+
+    def process_dataset_entry(self, data_entry):
+        input_values = os.path.splitext(data_entry[self.input_field])[0].split("/")
+        
+        data_entry[self.output_field] = input_values[-1]
+        return [DataEntry(data=data_entry)]
+
+
+class MakeTsv(BaseProcessor):
+    """
+    Processor for converting a JSON manifest file to a TSV (Tab-Separated Values) file.
+
+    Args:
+        **kwargs: Additional keyword arguments to be passed to the base class `BaseProcessor`.
+
+    Note:
+        This class inherits from the `BaseProcessor` class and provides functionality to convert a JSON manifest file to a TSV file.
```

Comment:
I don't think this note adds any extra information. The fact that it inherits from the class is clear in the class definition, and the info about its functionality is already present in the docstring. I think it's better to remove this note and other similar redundant notes in other docstrings. This would mean that more important notes would get more attention.

Reply:
Removed